201-vmss-azureautomationdsc/azuredeploy.json

```diff
@@ -0,0 +1,739 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "domainNameLabel": {
```

Comment:
declined, aligning with API name "label" rather than suggested "prefix" for API consistency